src/js/main/index.js

```diff
@@ -306,12 +306,67 @@ const statusChangeCallback = (response) => {
  * Autocomplete Part
  */
 $(function(){
+    // default autocomplete:
+    // fetch the newest job titles and company names
+    let default_job_titles = [];
+    let default_company_names = [];
+    $.ajax({
+        url: WTS.constants.backendURL + 'workings/latest',
+        dataType:"json",
+    }).done(function(res){
+        default_job_titles =
+            Array.from(new Set(
+                $.map(res.workings, (item,i)=>({
+                    "value": item.job_title,
+                }))
+            )).slice(0,4);
+        default_company_names =
+            Array.from(new Set(
+                $.map(res.workings, (item,i)=>({
+                    "value": item.company.name,
+                    "id": item.company.id,
```

Comment:
due to L429 `nameList.push({value: item.name, company_id: item.id});`, the `id: item.company.id` in L329 should be `company_id: item.company.id`